bazel/grpc_build_system.bzl

```diff
@@ -137,6 +139,32 @@ def grpc_proto_library(
         use_external = use_external,
         generate_mocks = generate_mocks,
     )
+def ios_cc_test(
+        name,
+        tags = [],
+        **kwargs):
+    ios_test_adapter = "//third_party/objective_c/google_toolbox_for_mac:GTM_GoogleTestRunner_GTM_USING_XCTEST";
+
+    apple_test_lib = name + "_test_lib_apple"
+    apple_tags = tags + ["manual", "ios_cc_test"]
+    if not any([t for t in tags if t.startswith("no_test_ios")]):
+        native.objc_library(
+            name = apple_test_lib,
+            srcs = kwargs.get("srcs"),
```

Comment:
Rather than doing `kwargs.get`, I wonder if you can just pass kwargs as a parameter to `objc_library`

Reply:
I guess you meant something like this?
```
        native.objc_library(
           name = apple_test_lib_ios,
           tags = apple_tags,
           alwayslink = 1,
            testonly = 1,
            **kwargs
        )
```
That does't work because [objc_library](https://docs.bazel.build/versions/master/be/objective-c.html#objc_library) doesn't take dict param.